products/BellHybrid/CMakeLists.txt

```diff
@@ -142,14 +142,14 @@ download_asset_release_json(json-common-target
                             ${CMAKE_CURRENT_SOURCE_DIR}/assets/assets_common.json
                            ${SYSROOT_PATH}/system_a/
                             MuditaOSPublicAssets
-                            0.0.15
+                            0.0.16
```

Comment:
Please make sure to update this value to 0.0.17
Keeping it that way may introduce conflicts and unexpected errors :)

Reply:
sure